backend/IntakeForms/GetIntakeForms.ts

```diff
@@ -0,0 +1,79 @@
+import { sequelize } from "../DB.js";
+import s from "sequelize";
+import { router } from "../Router.js";
+import { param, validationResult } from "express-validator";
+import { invalidRequest, notFound } from "../Utils/EndpointResponses.js";
+
+router.get("/api/intake-forms", async (req, res) => {
+  const intakeForms = await sequelize.models.IntakeForm.findAll();
+  res.send({
+    intakeForms,
```

Comment:
I honestly am not a huge fan of how sequelize includes capital case object keys in JSON. Also, I might have named `pageChildren` to be `intakeItems` so that other intake items that contain children intake items (like sections) have the same key as pages for their children. Additionally, it seems unnecessary to me for each intake item to send `intakeFormId` since the requester already asked only for intake items for that intake form.

I'm not sure how much of that could be customized in the sequelize up migrations, or if addressing it would require implementing a transformer to convert the internal sequelize representation to an external json api representation. I don't want to cause you a ton of work needlessly by insisting on having such a transformation layer - what are your thoughts?